data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleBuilder.java

```diff
@@ -211,11 +214,29 @@ private static Filter getFilter(DataPlaneContract.DialectedFilter filter) {
     };
   }
 
+  private WebClientOptions createWebClientOptionsFromCACerts(String CACerts) {
+    if (!CACerts.isEmpty()) {
+      return consumerVerticleContext
+        .getWebClientOptions()
+        .setPemTrustOptions(
```

Comment:
I believe we're modifying a shared object (WebClientOptions) and by doing that we're only configuring the client to trust whatever CA Certs we configure last, a more safe approach would be to consider WebClientOptions as an immutable object, so I'd copy/clone it (there should a constructor for that) and _then_ set the trust options.